ajax/libs/hyperHtml/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "hyperHtml",
+  "filename": "min.min.js",
+  "description": "A Fast & Light Virtual DOM Alternative",
+  "license": "ISC",
+  "author": "Andrea Giammarchi",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/WebReflection/hyperhtml.git"
+  },
+  "keywords": [
+    "dom",
+    "diff",
+    "performance",
+    "template",
+    "literals",
+    "lightweight",
+    "fast",
+    "react",
+    "virtual",
+    "lit-html",
+    "alternative"
+  ],
+  "npmName": "hyperhtml",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
```

Comment:
```suggestion
    {
      "basePath": "",
      "files": [
        "*.js"
      ]
    }
```

This appears to no longer be valid for more recent versions -- it would seem the required files are now at the root of the package:

```
tree node_modules/hyperhtml                                                                                                                                                            ✔  9023  13:15:19 
node_modules/hyperhtml
├── CHANGELOG.md
├── CONTRIBUTING.md
├── HELPERS.md
├── LICENSE.txt
├── README.md
├── cjs
│   ├── classes
│   │   └── Component.js
│   ├── hyper
│   │   ├── render.js
│   │   └── wire.js
│   ├── index.js
│   ├── objects
│   │   ├── Intent.js
│   │   └── Updates.js
│   ├── package.json
│   └── shared
│       ├── constants.js
│       └── utils.js
├── esm
│   ├── classes
│   │   └── Component.js
│   ├── hyper
│   │   ├── render.js
│   │   └── wire.js
│   ├── index.d.ts
│   ├── index.js
│   ├── objects
│   │   ├── Intent.js
│   │   └── Updates.js
│   └── shared
│       └── constants.js
├── esm.js
├── index.d.ts
├── index.js
├── logo.txt
├── min.js
├── no.js
├── package.json
├── rollup.ie.js
├── umd.d.ts
└── umd.js
```

Reply:
The only source of truth in any of my modules is the `esm/` folder. All scripts are generated from that folder, but the file that should be served via CDNs is the one specified in the `"unpkg": "min.js"` field. The `min.js` is already babelized and minified, and that's indeed what https://unpkg.com/hyperhtml serves to have a global `hyperHTML` reference.

To use the ESM variant, https://unpkg.com/hyperhtml?module points at the ESM entry point, which is the `esm/index.js`, but it also resolves automatically all imported sub-modules.

I hope this helps.

Reply:
@WebReflection - So we should bring in `min.js` and then all the files contained within `esm`?

Reply:
@MattIPv4 the `min.js` is the bundle already minified, the `index.js` is the same bundle pre-minification. `esm/index.js` is for native ESM modules, but it requires some resolution of itse dependencies, so if CDNJS cannot do such resolution, being static, `min.js` and `index.js` is all it should have, excluding everything else.

Reply:
P.S. worth saying *maybe* `umd.js` could be included too, for weird/edge cases.

Reply:
We support ts, so I'll include the typings as well.